react-payments/src/component/CardSlider/CardSlider.tsx

```diff
@@ -0,0 +1,75 @@
+import Slider from "component/common/Slider/Slider";
+import { API_URL } from "constants/index";
+import useFetch from "hooks/useFetch";
+import { CardDataContext, initCardDataAction } from "provider/CardDataProvider";
+import { useContext, useEffect, useRef, useState } from "react";
+import Card from "component/common/Card/Card.component";
+import SkeletonCardBox from "component/common/SkeletonCard/SkeletonCard.component";
+import PaymentLabel from "component/PaymentLabel/PaymentLabel";
+import CardAddLink from "component/CardAddLink/CardAddLink.component";
+
+function CardSlider() {
+  const { data, loading } = useFetch(`${API_URL}/api/cards`);
+  const cardDataContext = useContext(CardDataContext);
+  if (!cardDataContext) throw new Error("Cannot find cardDataContext");
+
+  const { cardData, dispatch } = cardDataContext;
+
+  const [currentSlide, setCurrentSlide] = useState(0);
+  const slideRef = useRef<HTMLDivElement | null>(null);
+
+  const nextSlide = () => {
+    if (currentSlide >= Object.keys(cardData).length) {
+      return;
+    } else {
+      setCurrentSlide(currentSlide + 1);
+    }
+  };
+
+  const prevSlide = () => {
+    if (currentSlide === 0) {
+      return;
+    } else {
+      setCurrentSlide(currentSlide - 1);
+    }
+  };
+
+  useEffect(() => {
+    if (slideRef.current !== null) {
+      slideRef.current.style.transition = "all 0.5s ease-in-out";
+      slideRef.current.style.transform = `translateX(calc(-${currentSlide}00% - ${
+        currentSlide * 80
+      }px))`;
+    }
+  }, [currentSlide]);
```

Comment:
```suggestion
  const { current, goNext, goPrev, slideRef } = useSlide();
```
hook으로 모으면 정말 깔끔해질 수 있겠는데요!